src/components/NewsletterForm/NewsletterForm.tsx

```diff
@@ -74,22 +105,24 @@ export const NewsletterForm = ({ className }: NewsletterFormProps) => {
     [`${className}`]: className
   });
 
+  // Show the success response if the submission has been sent
+  if (responseSuccess) {
```

Comment:
Minor: Perfectly valid to render this way however since this is React the ternary approach might be more consistent

i.e. 
```
return (
  {responseSuccess
    ?
    <response/>
    :
    <form />
  }
)

Reply:
Fair point - does this mean we aim to exclusively use ternary operators, then?

Reply:
Certainly for react component render (since that's what the React docs state should be used) but I don't think we should aim for it exclusively. In instances where code is made less readable then an `if` or `if else` or even refactoring would probably be better

Reply:
had a brief look at the react docs after you posted, but all I could find was:

> Just like in JavaScript, it is up to you to choose an appropriate style based on what you and your team consider more readable.
> - https://reactjs.org/docs/conditional-rendering.html#inline-if-else-with-conditional-operator (beneath code snippets)

tbh I'm happy with whatever is most readable too so it's not really an issue, more just curious as to if React do say "use this way" somewhere!

Reply:
Fair enough. I couldn't find the reference either :D